### DISCUSSION TITLE
Adding option to Turn off Climate if room temperature is too hot

### DISCUSSION THREAD

**Discussion by skyderman** [jmcollin78/versatile_thermostat#393] [Ideas]
I use Vtherm with climate ( AC ).

Sometime, the temperature compensation is not enought, and it will be great if we can set the AC to off, if Vtherm set AC's temperature at min and the room temperature still too high, then after X time, put the AC to off.

In my case, 
Vtherm is set on "Heat" at 18°
the room temp is 21.22°
the AC is set to 17.5 °(internal temp of AC is 23.6° )

But AC still produce heat ( even at min temp on the AC ) .
 
The only way to stop heating is to off the AC.

I will be great if we can do this with vTherm 

Thank you in advance

**Comment by jmcollin78** [comment]
Hello, this have been many times asked, but I won't do that. That will interfer with the user wishes, with the window open detection. See the answers I have already made to the same request. Thank you.
